index.bs

```diff
@@ -384,6 +384,56 @@ scheme, is encoded in the <code>URL</code> field.
   };
 </pre>
 
+<div class="example">
+In this example we build a page supporting a device. To start with when the page
+is loaded it should first check if the device is already plugged in and it has
+permission to access it by calling {{USB/getDevices()}},
+
+<pre>
+  document.addEventListener('DOMContentLoaded', async () => {
+    let devices = await navigator.usb.getDevices();
+    devices.forEach(device => {
+      // Add |device| to the UI.
+    });
+  });
+</pre>
+
+After the page is loaded the user may <a>connect</a> or <a>disconnect</a> a
+device from their system so script should also register for these events in
+order to keep the interface up-to-date,
+
+<pre>
+  navigator.usb.addEventListener('connect', device => {
+    // Add |device| to the UI.
+  });
+
+  navigator.usb.addEventListener('disconnect', device => {
+    // Remove |device| from the UI.
+  });
+</pre>
+
+If this is the first time the user has visited the page then there will be no
```

Comment:
"then it won't have permission to access any devices"? Using "permission" as an object tends to cause ambiguities.